### PR TITLE
General updates to URLs

### PR DESCRIPTION
**Pull request by Pilargit12**
Updated needed for the Platform Product Registry URL and index.js showing duplicate links

### CODE REVIEW

File: src/docs/automation-and-resiliency/openshift-project-resource-quotas.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ New project sets provisioned in **all clusters** of the BC Gov Private Cloud Paa
 
 **Note**: The Platform Services team may approach your team to discuss a downgrade in your quota if they find your application isn't using all of the resources within the current quota.
 
-If the default allocations aren't sufficient for your application, [you can ask for a quota increase](/request-quota-increase-for-openshift-project-set/). You'll need a Sysdig dashboard that shows that your application needs more of a specific resource type (CPU, RAM or storage) in a specific namespace. Provide this proof to the Platform Services team before they can approve a quota increase that you submit in the [Platform Project Registry](https://registry.developer.gov.bc.ca/public-landing).
+If the default allocations aren't sufficient for your application, [you can ask for a quota increase](/request-quota-increase-for-openshift-project-set/). You'll need a Sysdig dashboard that shows that your application needs more of a specific resource type (CPU, RAM or storage) in a specific namespace. Provide this proof to the Platform Services team before they can approve a quota increase that you submit in the [Platform Product Registry](https://registry.developer.gov.bc.ca/login).
```
Comment (w8896699) [inline src/docs/automation-and-resiliency/openshift-project-resource-quotas.md:30]: I dont think registry has login page

Comment (w8896699) [reply src/docs/automation-and-resiliency/openshift-project-resource-quotas.md:30]: My bad, now it get redirect...But I still think just `https://registry.developer.gov.bc.ca/` would be good enough, what you think @ShellyXueHan

Comment (ShellyXueHan) [reply src/docs/automation-and-resiliency/openshift-project-resource-quotas.md:30]: either way works fine, but if we are just referencing the product registry, might be better to using `https://registry.developer.gov.bc.ca/` without the subpath, incase our developers decide to change it later on then it'll throw a 404 ;)

File: src/docs/security-and-privacy-compliance/vault-secrets-management-service.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ Vault has built-in support for secret revocation. Vault can revoke not only sing
 
 ## Eligibility and prerequisites
 
-Your team is provisioned a Vault service account for each environment (dev, test, prod, and tools) through the automation backing the [Platform Services Registry](https://registry.developer.gov.bc.ca/public-landing). Each ProjectSet created in the registry can have up to two technical contacts associated with it. These technical contacts are given write access to Vault and can manage secrets within a ProjectSet's two mount points (nonprod, prod).
+Your team is provisioned a Vault service account for each environment (dev, test, prod, and tools) through the automation backing the [Platform Product  Registry](https://registry.developer.gov.bc.ca/login). Each ProjectSet created in the registry can have up to two technical contacts associated with it. These technical contacts are given write access to Vault and can manage secrets within a ProjectSet's two mount points (nonprod, prod).
```
Comment (ShellyXueHan) [inline src/docs/security-and-privacy-compliance/vault-secrets-management-service.md:62]:
```suggestion
Your team is provisioned a Vault service account for each environment (dev, test, prod, and tools) through the automation backing the [Platform Product Registry](https://registry.developer.gov.bc.ca/login). Each ProjectSet created in the registry can have up to two technical contacts associated with it. These technical contacts are given write access to Vault and can manage secrets within a ProjectSet's two mount points (nonprod, prod).
```

File: src/docs/automation-and-resiliency/openshift-project-resource-quotas.md (hunk 1 above)
Comment (Pilargit12) [reply src/docs/automation-and-resiliency/openshift-project-resource-quotas.md:30]: Perfect, thank you team, I will update it to make it more clear! Thank you both so much!

**Pull request merged**